collector/systemd_linux.go

```diff
@@ -112,6 +113,11 @@ func NewSystemdCollector(logger log.Logger) (Collector, error) {
 		"Whether the system is operational (see 'systemctl is-system-running')",
 		nil, nil,
 	)
+	systemShutdownDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "system_shutdown_timestamp"),
+		"Time for a scheduled shutdown (see 'systemctl status systemd-shutdownd.service')",
```

Comment:
I'm also not sure how to represent the "no shutdown scheduled" state. in my script, i used "zero seconds" as a value for that, but the property returned somehow uses something else (which looks a lot like MAX_INT-1, AKA 2^64-1, AKA 18446744073709551615 ≈ 1,844 674 407 4 × 10^19)

also note that on my laptop, this morning, after the device went to sleep on its own after a timeout, dbus says this:

```
anarcat@angela:~$ busctl get-property org.freedesktop.login1 /org/freedesktop/login1 org.freedesktop.login1.Manager ScheduledShutdown
(st) "suspend" 0
```